code/modules/shuttle/medisim.dm

```diff
@@ -11,6 +11,9 @@
 
 /obj/machinery/capture_the_flag/medisim/Initialize(mapload)
 	. = ..()
+	// Ensure that the historical simulation is extra bloody.
+	player_traits -= TRAIT_NEVER_WOUNDED
```

Comment:
? shouldnt you just set player_traits to a list containing nothing in the definition

Reply:
That would have the same effect at this time, but there may be additional traits added to CTF players in the future, and I wouldn't want to accidentally deprive the medival CTF of those traits.

Reply:
It's better if it's set on the object.
If someone adds those additional traits they can add them to the children as well, and should not assume all of them will inherit that list.